### ISSUE TITLE
We need to convert nested asciidoc lists to markdown (double asterisks need to be converted to markdown syntax for nested lists)

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1782]
```
- 1-level list item
- 1-level list item
** 2-level list item
```
&nbsp;needs to become
```
- 1-level list item
- 1-level list item
  * 2-level list item
```

**Comment by ds26gte** [comment]
Please note that you are allowed to use the Markdown-style 2-level bullet in AsciiDoc too, in which case no explicit translation to Markdown is required.

**Comment by schanzer** [comment]
Closed by 72e0822fd0453b2969e6c28baaa0197074594454 and f571351b72745997b9dea6c5cdc8dc7842d72c08
